### ISSUE TITLE
All Video Content

### ISSUE THREAD

**Issue by kimhenryWAM** [wam-higgins/wam-ipad#23]
All video content: should be housed on iPad rather than mapped to YouTube.  Need to address accessibility of captioning (which is currently addressed via YouTube).  There should be a function allowing enlargement of caption font-size and then return to default setting.

**Comment by bchabot** [comment]
We should review what we have, whether it is a placeholder or final content.

Brian to define renaming files for future updates:

ie:

BTS1.mp4
BTS1-clip.mp4
etc.
